docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -0,0 +1,912 @@
+---
+lang: en
+title: 'Authentication Tutorial'
+keywords: LoopBack 4.0, LoopBack 4, Authentication, Tutorial
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/authentication-tutorial.html
+summary: A LoopBack 4 application that uses authentication
+---
+
+## Overview
+
+LoopBack 4 has an authentication package `@loopback/authentication` which allows
+you to secure your API endpoints with custom authentication strategies that
+implement a common interface and are registered via an extensionPoint/extension
+pattern.
+
+## Try it out
```

Comment:
It's a bit out of context to have `Try it out` here. At least something like `Try out authentication in an example`.

Reply:
Maybe we should add some more information before try it out.

Reply:
@raymondfeng 

>It's a bit out of context to have Try it out here. At least something like Try out authentication in an example. 

All the tutorials have a section title `Try it out`.

Reply:
@raymondfeng 

> Maybe we should add some more information before try it out. 

I added a section before `Try it out`.